---
layout: default
title: Pathogen Surveillance Training Curricula
permalink: /courses/curricula/
---

<head>

    <!-- CSS -->
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.min.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/buttons/1.7.0/css/buttons.dataTables.min.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/select/1.3.3/css/select.dataTables.min.css">
    
    <!-- JavaScript -->
    <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.min.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.7.0/js/dataTables.buttons.min.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/select/1.3.3/js/dataTables.select.min.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.7.0/js/buttons.html5.min.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.7.0/js/buttons.print.min.js"></script>
    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.7.0/js/buttons.colVis.min.js"></script>
    
    <!-- Required for Excel export -->
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.3/jszip.min.js"></script>
    
    <!-- Required for PDF export -->
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.53/pdfmake.min.js"></script>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.53/vfs_fonts.js"></script>

</head>


<div class="table-responsive-sm" class="table-wrapper" >
<table id="example" class="table table-striped table-bordered">


  <thead>
      <tr>
          <th> Module Code</th>
      
          <th>Pathogen</th>
          <th>Persona </th>
          <th name="title_column">Module Title</th>

      </tr>
  </thead>
  <tbody>

    <!-- bioinfomatics foundational -->
    <tr>
      <td>001</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td id="title_column" >Unix/linux, command line, shell scripting </td>
   
  

    </tr>
    <tr>
      <td>002</td>

      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td id="title_column">Data analytics</td>
  


    </tr>
    <tr>
      <td>003</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>Phylogenetics</td>
     

    
    </tr>
    <tr>
      <td>004</td>
  
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>Informatics skills </td>
     


    </tr>
    <tr>
      <td>005</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>Workflows and containerization</td>


  
    </tr>
    <tr>
      <td>006</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>Programming in Python</td>
      

  
    </tr>
    <tr>
      <td>007</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>R programming</td>
  

  
    </tr>
    <tr>
      <td>008</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>Github and version control</td>

 
    </tr>
    <tr>
      <td>009</td>
      <td>Foundational</td>
      <td>Bioinformatics Scientist</td>
      <td>Ontologies, standards, data curation and submission</td>

 
    </tr>
   <!-- bioinfomatics foundational -->
  <!-- bioinfomatics viral -->
  <tr>
    <td>001</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Introduction to NGS data processing</td>


  </tr>
  <tr>
    <td>002</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>NGS, Illumina, ONT workflows</td>


  </tr>
  <tr>
    <td>003</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Nextclade and Pangolin</td>


  </tr>
  <tr>
    <td>004</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Galaxy workflow (or other slected workflow(s))</td>


  </tr>
  <tr>
    <td>005</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Phylogenetics</td>


  </tr>
  <tr>
    <td>006</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Advanced bioinformatics analysis for viruses</td>


  </tr>
  <tr>
    <td>007</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Data curation and submission</td>

 
  </tr>

  <tr>
    <td>008</td>
    <td>Viral</td>
    <td>Bioinformatics Scientist</td>
    <td>Dissemination of genomic surveillance findings <br> to policy makers and the public 
    </td>


  </tr>

  <!-- Bioinformatics bacterial -->
  <tr>
    <td>001</td>
    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>Introduction to bacterial NGS data processing</td>


  </tr>
  <tr>
    <td>002</td>

    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>Bacterial pathogen data analysis</td>


  </tr>
  <tr>
    <td>003</td>
    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>PATRIC 
    </td>


  </tr>
  <tr>
    <td>004</td>
    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>Phylogenetics</td>


  </tr>
  <tr>
    <td>005</td>
    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>Metagenomics for pathogen surveillance</td>


  </tr>
  <tr>
    <td>006</td>
    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>Data curation and submission</td>


  </tr>
  <tr>
    <td>007</td>
    <td>Bacterial</td>
    <td>Bioinformatics Scientist</td>
    <td>Dissemination of genomic surveillance findings <br> to policy makers and the public 
    </td>


  </tr>

    <!-- bioinfomatics bacterial -->

    <!-- bioinformatics Eukaryotic-->
    <tr>
      <td>001</td>

      <td>Eukaryotic</td>
      <td>Bioinformatics Scientist</td>
      <td>Data analysis, interpretation, and presentation</td>

    </tr>
    <tr>
      <td>002</td>

      <td>Eukaryotic</td>
      <td>Bioinformatics Scientist</td>
      <td>Using data for decision-making, data sources for Malaria</td>


    </tr>
    <tr>
      <td>003</td>
      <td>Eukaryotic</td>
      <td>Bioinformatics Scientist</td>
      <td>Phylogenetics
      </td>


    </tr>
    <tr>
      <td>004</td>
      <td>Eukaryotic</td>
      <td>Bioinformatics Scientist</td>
      <td>Data curation and submission</td>


    </tr>


      <!-- bioinformatics Eukaryotic-->

      <!-- bioinformatics fungal-->

      <!-- bioinformatics fungal-->

    <!--Wetlab foundational-->
    <tr>
      <td>001</td>
      <td>Foundational</td>
      <td>Wet-Lab Scientist only</td>
      <td>Health and safety protocols</td>

    </tr>
    <tr>
      <td>002</td>
      <td>Foundational</td>
      <td>Wet-Lab Scientist only</td>
      <td>Lab practical: DNA/RNA sequencing</td>


    </tr>

    <!--Wetlab foundational-->


    <!--Wetlab Viral-->
    <tr>
      <td>001</td>
      <td>Viral</td>
      <td>Wet-Lab Scientist only</td>
      <td>Lab practical: RNA/DNA sequencing
      </td>

    </tr>
    <tr>
      <td>002</td>
      <td>Viral</td>
      <td>Wet-Lab Scientist only</td>
      <td>ARTIC amplicon sequencing</td>


    </tr>
       <!--Wetlab Viral-->


           <!--Wetlab Bacterial-->
    <tr>
      <td>001</td>
      <td>Bacterial</td>
      <td>Wet-Lab Scientist only</td>
      <td>Health and safety protocols
      </td>

    </tr>
    <tr>
      <td>002</td>
      <td>Bacterial</td>
      <td>Wet-Lab Scientist only</td>
      <td>Sample processing and library preparation <br> (Theory and wet lab practical)</td>


    </tr>


    <tr>
      <td>002</td>
      <td>Bacterial</td>
      <td>Wet-Lab Scientist only</td>
      <td>Lab practical: DNA sequencing</td>


    </tr>
       <!--Wetlab Bacterial-->
      <!--Wetlab Eukaryotic-->
    <tr>
      <td>001</td>
   
      <td>Eukaryotic</td>
      <td>Wet-Lab Scientist only</td>
      <td>Health and safety protocols</td>

    </tr>

    <tr>
      <td>002</td>
      <td>Eukaryotic</td>
      <td>Wet-Lab Scientist only</td>
      <td>Sample processing and library preparation <br> (Theory and wet lab practical)</td>


    </tr>

    <tr>
      <td>003</td>
      <td>Eukaryotic</td>
      <td>Wet-Lab Scientist only</td>
      <td>Lab practical: DNA sequencing</td>


    </tr>

    <!--Wetlab Eukaryotic-->

        <!--Wetlab fungal-->
        <!-- <tr>
          <td>001</td>
          <td>Fungal</td>
          <td>Wet-Lab Scientist only</td>
          <td>Health and safety protocols</td>
    
        </tr>
        <tr>
          <td>002</td>
          <td>Fungal</td>
          <td>Wet-Lab Scientist only</td>
          <td>Sample processing and library preparation <br>(Theory and wet lab practical)</td>
    
    
        </tr>

        <tr>
          <td>002</td>
          <td>Fungal</td>
          <td>Wet-Lab Scientist only</td>
          <td>Lab practical: DNA sequencing</td>
    
    
        </tr> -->
    
        <!--Wetlab fungal-->
    <!--Wetlab/Drylab Foundational-->
          <tr>
            <td>001</td>
            <td> Foundational</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Ethics and policies</td>
      
          </tr>


          <tr>
            <td>002</td>
            <td> Foundational</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Study design </td>
      
      
          </tr>

          <tr>
            <td>003</td>
            <td> Foundational</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Dissemination of results</td>
      
      
          </tr>
      <!--Wetlab/Drylab Foundational-->

          <!-- Wetlab/Drylab viral-->
          <tr>
            <td>001</td>
            <td>Viral</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Introduction to genomic epidemiology </td>
      
      
          </tr>
      
          <tr>
            <td>002</td>
            <td>Viral</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Study design for viral pathogen surveillance</td>
      
      
          </tr>


          <tr>
            <td>003</td>
            <td>Viral</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Introduction to NGS </td>
      
      
          </tr>
      
    
           <!--Wetlab-drylab Bacterial-->
           <tr>
            <td>001</td>
            <td>Bacterial</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Introduction to genomic epidemiology 
            </td>
      
          </tr>
          <tr>
            <td>002</td>
            <td>Bacterial</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Study design for bacterial pathogen surveillance</td>
      
      
          </tr>
      
      
          <tr>
            <td>002</td>
            <td>Bacterial</td>
            <td>Wet-lab and Dry-lab Scientist</td>
            <td>Introduction to NGS </td>
      
      
          </tr>
             <!--Wetlab drylab Bacterial-->

      <!--Wetlab/Drylab Eukaryotic-->
      <tr>
        <td>001</td>
        <td>Eukaryotic</td>
        <td>Wet-lab and Dry-lab Scientist</td>
        <td>Introduction to Surveillance, Monitoring, and Evaluation.</td>

      </tr>
  
     <!--Wetlab/Drylab Eukaryotic-->


      <!--Wetlab/Drylab fungal-->
      <!-- <tr>
        <td>001</td>
        <td>Fungal</td>
        <td>Wet-lab and Dry-lab Scientist</td>
        <td>Health and safety protocols</td>
  
      </tr>
      <tr>
        <td>002</td>
        <td>Fungal</td>
        <td>Wet-lab and Dry-lab Scientist</td>
        <td>Study design for fungal pathogen surveillance</td>
  
  
      </tr>

      <tr>
        <td>002</td>
        <td>Fungal</td>
        <td>Wet-lab and Dry-lab Scientist</td>
        <td>Introduction to NGS</td>
  
  
      </tr> -->
  
      <!--Wetlab/Drylab  fungal-->






  </tbody>
  <tfoot>
      <tr>
        <th>Code</th>
      
        <th>Pathogen</th>
        <th>Persona </th>
        <th class="column_title">Title</th>
  

      </tr>
  </tfoot>
</table>


</div>
<script>
$(document).ready(function() {
    function initializeTable(tableId, columnSelectId) {
        var table = $(tableId).DataTable({
            dom: 'lBfrtip',
            buttons: [
                {
                    extend: 'collection',
                    text: 'Export',
                    buttons: [
                        {
                            extend: 'copy',
                            exportOptions: {
                                columns: ':visible'
                            },
                            action: function(e, dt, button, config) {
                                var recordCount = dt.page.len();
                                config.exportOptions.rows = ':lt(' + recordCount + ')';
                                $.fn.dataTable.ext.buttons.copyHtml5.action.call(this, e, dt, button, config);
                            }
                        },
                        {
                            extend: 'excel',
                            exportOptions: {
                                columns: ':visible'
                            },
                            action: function(e, dt, button, config) {
                                var recordCount = dt.page.len();
                                config.exportOptions.rows = ':lt(' + recordCount + ')';
                                $.fn.dataTable.ext.buttons.excelHtml5.action.call(this, e, dt, button, config);
                            }
                        },
                        {
                            extend: 'csv',
                            exportOptions: {
                                columns: ':visible'
                            },
                            action: function(e, dt, button, config) {
                                var recordCount = dt.page.len();
                                config.exportOptions.rows = ':lt(' + recordCount + ')';
                                $.fn.dataTable.ext.buttons.csvHtml5.action.call(this, e, dt, button, config);
                            }
                        },
                        {
                            extend: 'pdf',
                            exportOptions: {
                                columns: ':visible'
                            },
                            action: function(e, dt, button, config) {
                                var recordCount = dt.page.len();
                                config.exportOptions.rows = ':lt(' + recordCount + ')';
                                $.fn.dataTable.ext.buttons.pdfHtml5.action.call(this, e, dt, button, config);
                            }
                        }
                    ]
                },
                'colvis'
            ],
            columnDefs: [{
                targets: '_all',
                visible: true // Display all columns by default
            }]
        });

        // Create checkboxes for column selection
        table.columns().every(function(index) {
            var column = this;
            var checked = column.visible() ? 'checked' : '';
            var checkbox = $('<label><input type="checkbox" ' + checked + '>' + $(column.header()).text() + '</label>');
            
            // Handle checkbox click event
            checkbox.find('input').on('click', function() {
                var isChecked = this.checked;
                column.visible(isChecked);
            });
            $(columnSelectId).append(checkbox);
        });
    }

    initializeTable('#example', '#columnSelect1');
    // initializeTable('#example3', '#columnSelect2');
});




</script>

