---
layout: default
title: "Introduction to Bioinformatics Analysis of Plasmodium Falciparum Genome data training 2022"
date: 2022-10-28
main_menu: courses
permalink: /:year/:title
category: upcoming
course_overview : Genomic Epidemiology of resistant Plasmodium Falciparum in Côte d’Ivoire with the aim to uncover the genomic signatures due to various selection pressures to be identified (host immune system? background genetics? epidemiological facies? or other).To understand the spread over the country of populations of parasites carrying any malaria drug resistant-conferring markers (the withdrawn ones as well as the ones on official uset).
  
start_date: 2022-11-28
end_date: 2022-12-02
group: group1
---

<!-- ### SARS-CoV-2 NGS bioinformatics course 2021 -->
The NGS Academy forms part of the Africa CDC Pathogen Genomics Initiative (PGI). This initiative is funded by the Bill & Melinda Gates Foundation. Our training initiatives are carried out in partnership with the African Society for Laboratory Medicine (ASLM).
<hr>

<!-- <b>Course outline:</b> [ SARS-CoV-2 NGS Bioinformatics Course 2021]({{ site.url }}/docs/course_outline.pdf) -->

<p align="left"><b >Course overview:</b></p>

 <p id="description" align="left">Genomic epidemiology of resistant P. falciparum in Côte d’Ivoire with the aim to uncover the genomic signatures due to various selection pressures to be identified (host immune system? background genetics? epidemiological facies? or other).
To understand the spread over the country of populations of parasites carrying any malaria drug resistant-conferring markers (the withdrawn ones as well as the ones on official uset). Training with a view to the proper understanding of implementing NGS sequencing (our institute was endowed with a NextSeq 550).</p>

<p><b>The course will cover:</b></p>


<div class="row">
    <div class="col-sm-2">
        <div><img alt="..." src="/ngs-academy/img/py.png" width="150%" />
            <!-- <p class="small"> Source: CBIO UCT</p> -->
        </div>
    </div>
    <div class="col-sm-2">
        <div><img alt="..." src="/ngs-academy/img/r.jpg" width="60%" />
            <!-- <p class="small"> Source: CBIO UCT</p> -->
        </div>
    </div>
    <div class="col-sm-2">
        <div><img alt="..." src="/ngs-academy/img/git.png" width="100%" />
            <!-- <p class="small"> Source: CBIO UCT</p> -->
        </div>
    </div>
    <div class="col-sm-2">
        <div><img alt="..." src="/ngs-academy/img/HPC.jpg" width="100%" />
            <!-- <p class="small"> Source: CBIO UCT</p> -->
        </div>
    </div>
         <div class="col-sm-2">
        <div><img alt="..." src="/ngs-academy/img/snakemake.png" width="100%" />
            <!-- <p class="small"> Source: CBIO UCT</p> -->
        </div>
    </div>

</div>


<p align="left"><b class="text-left">Pre-requisites and eligibility:</b></p>

<p>No prior bioinformatics knowledge is required.
</p>

<p align="left"><b class="text-left">Objectives and expected outcomes:</b>
The training has a double objective, to understand the genomic epidemiology of resistant P. falciparum in Côte d'Ivoire in order to discover the genomic signatures due to the different selection pressures to be identified (immune system of the host, genetic background, facies epidemiological, etc.). It is also a question of understanding the mode of diffusion and the distribution on the territory of the populations of parasites carrying markers conferring resistance to antimalarial drugs.
</p>

<p align="left"><b class="text-left">Intended audience:</b></p>

<p>Beginner - Intermediate  level  Bioinformaticians working with next-generation sequencing data in Africa.
</p>


<!-- test -->



<p><b>Logistics: </b></p>
<hr>

<ol class="list-unstyled">

 <li id="location"><b>Venue of course</b>: Institut Pasteur de Côte d'Ivoire (IPCI, Cote d'Ivore) </li>


<li id ="startDate"><b>Start date: </b>2022-11-28</li>

<li id="endDate"><b>End date: </b>2022-12-02</li>

<!-- <li><b>Course sponsors</b>: TBA</li> -->

<!-- <u><li><b>Participant applications</b></li></u>

<li><b>Registration for participants opened:</b> 25 Feb 2022</li> -->

<!-- <li><b>Registration for participants closed:</b> 04 March 2022</li> -->

<!-- <li><b>Notification date for successful Applicants:</b> </li> -->
</ol>

 <b>Course curriculum:</b>
<hr>
<ol class="list-unstyled">

<li>1. Malaria, NGS, software tools, snakemake. </li>
<li>2. Phylodynamics and phylogeography </li>
<li>3. Machine learning/modeling </li>
<li>4. Git, version control. QC/Sequence assembly </li>
<li>5. Workflows. Metagenomic analysis and Genome Analysis Pipeline</li>
<li>6. Snakemake & R programming.</li>
<li>7. Workflow management and Nextflow Conda environment</li>
</ol>
 <b>Practical work:</b>
<hr>
<ol class="list-unstyled">

<li>MSA and construction of phylogenetic trees </li>
<li>Advanced HPC usage and maintenance </li>
<li>Development and maintenance of bioinformatics tools</li>
<li>Implementing Machine Learning in Bioinformatics </li>
<li>Bash, Python and R scripting and version control.</li>
</ol>






